packages/react-ui-core/src/Carousel/CarouselNavigation.js

```diff
@@ -21,6 +21,11 @@ export default class CarouselNavigation extends PureComponent {
     theme: {},
   }
 
+  get tagItem() {
+    const direction = this.props.direction === 'previous' ? 'left' : 'right'
+    return `${direction}_arrow`
```

Comment:
Can this be overridden if needed?

Reply:
Yes... but why would you need to?

Reply:
![screen shot 2018-06-18 at 4 03 18 pm](https://user-images.githubusercontent.com/10748767/41566637-35626684-7311-11e8-8943-a9227494a067.png)

You should probably look at the code a little more on how things are done.  I can help explain though.  The structure for all react-ui components is generally setup up to spread the extra props over the component. That keeps for flexible / reusable code.  If you look at the above the spread props are _after_ the `data-tag_item` which means that if you were to pass `data-tag_item="foo"` in as a prop, the component would now have that on the `<button data-tag_item="foo">` DOM node since it gets passed through to `<Button />` and Button [spreads props](https://github.com/rentpath/react-ui/blob/master/packages/react-ui-core/src/Button/Button.js#L37) as well.